app/javascript/components/student/iterations-list/RepresenterFeedback.tsx

```diff
@@ -31,11 +31,13 @@ function EditedBy({
   author,
   editor,
 }: Pick<Props, 'author' | 'editor'>): JSX.Element | null {
-  if (editor && editor.name !== author.name) {
+  if (!editor || editor.name === author.name) {
+    return null
+  } else {
```

Comment:
You don't need this return now (that's the beauty of the early return is that everything else stops needing to be indented).